include/NovelRT/Maths/GeoVector.h

```diff
@@ -59,6 +59,14 @@ namespace NovelRT::Maths {
       _value.y = value;
     }
 
+    inline GeoVector<T> getNormalised() const noexcept {
+      return GeoVector<T>(glm::normalize(getVec2Value()));
+    }
+
+    inline float getMagnitude() const noexcept {
```

Comment:
nit: I think `length` is the more standard term

Reply:
I've only ever heard it addressed as magnitude in a game dev context, but I can change it if you wish.

Reply:
both glm and dxmath use the term `length` as do `glsl` and `hlsl`

Reply:
Added an alias for `getMagnitude`.